src/xenia/ui/imgui_drawer.h

```diff
@@ -16,8 +16,8 @@
 #include "xenia/ui/immediate_drawer.h"
 #include "xenia/ui/window_listener.h"
 
+struct ImGuiContext;
```

Comment:
Previous list was sorted alphabetically, this breaks it.

Reply:
sorted those forward declarations.